migration/incremental/006-tables.sql

```diff
@@ -39,11 +39,6 @@ GRANT SELECT ON TABLE _prom_catalog.series TO prom_reader;
 GRANT SELECT, INSERT, UPDATE, DELETE ON TABLE _prom_catalog.series TO prom_writer;
 
 
```

Comment:
I'd leave a comment here about the removed parent indexes.

Reply:
resolved in [356b1a6](https://github.com/timescale/promscale_extension/pull/185/commits/356b1a63a3987e60c00f6f0816beef2941224c03)